internal/querynode/impl.go

```diff
@@ -1422,13 +1565,23 @@ func (node *QueryNode) GetDataDistribution(ctx context.Context, req *querypb.Get
 func (node *QueryNode) SyncDistribution(ctx context.Context, req *querypb.SyncDistributionRequest) (*commonpb.Status, error) {
```

Comment:
Need to check the permissibility of this synchronization operation during the course of the migration process

Reply:
During the stopping process, this request can be responded normally, because this is mainly to synchronize the distribution on the querynode shardleader.